test/libsolidity/syntaxTests/constructor/interface_constructor_new.sol

```diff
@@ -1,7 +1,7 @@
 interface I {
-	constructor();
+	constructor() external;
```

Comment:
Well, arguably one could also use ``public`` here - or split into two test cases :). But I think it should be fine like this as well.

Reply:
OK, I just saw that changing from public to external was a previous review comment :smile:.

Reply:
Oh I'm sorry! Of course it should be public.

Reply:
Yes, ``public`` probably makes more sense actually.

Also it may make sense to give the constructor an empty implementation to silence the ``Constructor must be implemented if declared`` error that's not really related to this test.

Reply:
@ekpyron Giving the constructor an empty implementation would additionally report ``Functions in interfaces cannot have an implementation.``, which is also distracting from the actual test expectations.
I can clearly see some potential type checker improvements here ;)

Reply:
Ah yes - haven't thought about that :). It's a challenging test, since no matter what you do it fails in so many ways :).

Reply:
That's true! But actually we have a quite similar test for constructors being used in libraries. I'm wondering if we should consider making the file names more verbose in the future. But that's not in the scope of this PR ;)